File: C/Armstrong.c
#include<stdio.h>
void main()
{
	int w,n,d,s=0;
	printf("Enter a number : ");
	scanf("%d",&n);
	w=n;
	while(w!=0)
	{
		d=w%10;
		s=s+d*d*d;
		w/=10;
	}
	if(s==n)
	printf("Armstrong number");
	else
	printf("Not an Armstrong number");
}

File: Python/armstrong.py
def isArmstrong(n):
    s, w = 0, n
    while w != 0:
        s += (w % 10) ** 3
        w //= 10
    if s == n:
        return True
    else:
        return False

n = int(input("Enter a number: "))
if isArmstrong(n):
    print("Armstrong number")
else:
    print("Not an Armstrong number")